### ISSUE TITLE
Following readme example leads to propType warning

### ISSUE THREAD

**Issue by ahctangU** [davidhu2000/react-spinners#51]
**Do you want to request a feature or report a bug?**
I want to report a bug.

**If reporting a bug, what version of `react-spinners` are you currently using?**
I am using  react-spinners version 0.5.0

**What is the current behavior?**
Following the example in the read me causes a warning:
`Warning: Failed prop type: Invalid prop `css` of type `object` supplied to `Loader`, expected `string`.`

**If the current behavior is a bug, please provide the steps to reproduce**
Follow the code example in the read me.

**Comment by MahbbRah** [comment]
Same issue Here.

**Comment by wamujlb** [comment]
@davidhu2000 The problem is in this line: 

`css: PropTypes.string`, it should be `PropTypes.object`. Because as `css()`.

I can create a PR for it. But you didn't leave any guide about how to contribute to your project :)

**Comment by davidhu2000** [comment]
https://github.com/davidhu2000/react-spinners/blob/master/CONTRIBUTING.md

😄

**Comment by hel-hadi** [comment]
Nice. i'm waiting this PR :)

**Comment by rsolomonjr** [comment]
> @davidhu2000 The problem is in this line:
> 
> `css: PropTypes.string`, it should be `PropTypes.object`. Because as `css()`.
> 
> I can create a PR for it. But you didn't leave any guide about how to contribute to your project :)

Same.

**Comment by davidhu2000** [comment]
This issue has been resolved in `0.5.2`. 

`PropTypes.string` has been changed to 

```
PropTypes.shape({
  name: PropTypes.string,
  styles: PropTypes.string
})
```

**Comment by davidhu2000** [comment]
**correction**: I forgot to update the default value of `css` prop. This was resolved in `0.5.3`.
